Data-Engineering/Airflow/example_ezaf_spark_csv_to_parquet_fts.py

```diff
@@ -22,7 +22,6 @@
     tags=['e2e example', 'ezaf', 'spark', 'csv', 'parquet', 'fts'],
     params={
         'training_path': Param("financial-processed", type="string"),
-        's3_secret_name': Param("spark-s3-creds", type="string"),
```

Comment:
Why did you remove it? Do you want to leave secret name hardcoded in the SparkApplication yaml file?